.github/workflows/spm.yml

```diff
@@ -17,7 +17,13 @@ jobs:
     runs-on: macOS-latest
     steps:
     - uses: actions/checkout@v2
+    - name: Xcode 12
+      run: sudo xcode-select -s /Applications/Xcode_12_beta.app/Contents/Developer
     - name: Build
       run: swift build
-    - name: Run
-      run: swift test
+    - name: Initialize xcodebuild
+      run: xcodebuild -list
+    - name: Core ObjC Unit Tests macOS
+      run: xcodebuild -scheme CoreUnit test
+    - name: Core ObjC Unit Tests iOS
+      run: xcodebuild -scheme CoreUnit test -sdk iphonesimulator -destination 'platform=iOS Simulator,name=iPhone 11'
```

Comment:
We can change this to `xcodebuild -scheme Firebase-Package test ...` and it'll run all `testTarget`s specified in the manifest. Maybe that's enough to unblock it for now and we run less frequently or on changes to the manifest?

Then once we hear more about running individual targets we can split it up accordingly.

Reply:
Done. We can defer figuring out figuring out testing individual targets.